2018/002-update-channels.md

```diff
@@ -0,0 +1,130 @@
+ Title
+
+| Field  | Value      |
+|:-------|:-----------|
+| Status | Draft      |
+| Date   | 2018-07-31 |
+
+## Introduction
+
+### Problem description
+
+Updates can be disruptive for a customer's application stack (ex. Kubernetes
+upgrade from 1.10.x to 1.11.x, default pod security policy introduction).
+
+Right now there is no visibility of what is inside an update and furthermore
```

Comment:
Agreed we should present a summary with a link to a full changelog on major cluster upgrades. Maybe for now we can make use of zypper output - it prints packages that will change as far as I know.

Reply:
If we get a new channel, we can only see that there is a new channel, but not what it contains. As long as we have not switched the channel, we don't get the permissions.
For the "normal" updates, we can try to create a list of packages and provide them to velum.

Reply:
Good point on the new channel. There must be a better way besides crawl the repository from the location found in whatever.repo